src/App.tsx

```diff
@@ -83,11 +61,7 @@ export const App: React.FC = () => {
                 )}
 
                 {author && loaded && !hasError && posts.length > 0 && (
```

Comment:
Pls [see falsy values doc](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
                {author && loaded && !hasError && !!posts.length && (
```

Reply:
changed